### ISSUE TITLE
How to process when having multiple files from Facebook?

### ISSUE THREAD

**Issue by Lyro1** [Allexio/fb-user-data-analyser#1]
My Facebook data recording file is apparently too big, so Facebook splitted it in 3 separate zip files. 
Is there a way to import those 3 zips in the tool?

**Comment by Allexio** [comment]
Hey, I had no idea that Facebook would split the files when they are too big, as of right now, the tool does not support importing multiple zip files.
That being said it *should* work if you unzip the separate facebook files and then zip them back up yourself?
What does the file structure look like for the data you've downloaded?

**Comment by Lyro1** [comment]
I only got one at the moment (my connexion is lame), and it contains the `messages` folder which seems incomplete.
I will wait until I get all the parts, merge them in a single zip file, and come back with the result.

**Comment by Allexio** [comment]
Yeah, the messages folder is one of the bigger ones with photos_and_videos, because it contains all the received and sent photos via messenger.
Another maybe easier solution would be to download lower quality pictures. Especially if you have a slow internet connection.
Also, for the record, I'm glad that you're interested in the project :)

**Comment by Allexio** [comment]
FYI: for the main account I tested with, which had about 4 thousand pictures in messages, the nudity detection algo took about 15 minutes to process them.
So if you have many times that, it might become a problem...
